src/transformers/generation/utils.py

```diff
@@ -4610,6 +4602,13 @@ def assisted_decoding(
                 outputs, model_kwargs, is_encoder_decoder=self.config.is_encoder_decoder
             )
 
+            # Update attention_mask
```

Comment:
```suggestion
            # Update attention_mask for the assistant's next round of generations
```
(add some more information)